infrastructure/cdn-in-a-box/optional/vpn/Dockerfile

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 FROM ubuntu:18.04
-RUN apt-get update && apt-get install -y vim tree wget iptables openvpn iputils-ping net-tools dnsutils
+RUN apt-get update && apt-get --no-install-recommends install -y vim tree wget iptables openvpn iputils-ping net-tools dnsutils
```

Comment:
See above—I would prefer disabling `APT::Install-Recommends` and `APT::Install-Suggests` in the APT configuration to passing `--no-install-recommends` for every apt or apt-get command.